src/java.desktop/windows/classes/sun/awt/shell/Win32ShellFolder2.java

```diff
@@ -1412,7 +1418,7 @@ static class MultiResolutionIconImage extends AbstractMultiResolutionImage {
 
         public MultiResolutionIconImage(int baseSize, Map<Integer, Image> resolutionVariants) {
             assert !resolutionVariants.containsValue(null)
-                   : "There are null icons in the MRI variants map";
+                    : "There are null icons in the MRI variants map";
```

Comment:
Here's an extra space. The colon aligned with `!` which starts the condition.

Could you please revert it too?